testing-overhead/src/test/java/io/opentelemetry/containers/PostgresContainer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.containers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class PostgresContainer {
```

Comment:
any thoughts to using this thing?  https://www.testcontainers.org/modules/databases/postgres/

Reply:
Interesting. I hadn't seen that. I'll try it out and see how it goes...

Reply:
Seems ok. It doesn't provide a whole lot, but it does abstract out a few pg internals (like having to know the environment var names etc).  👍🏻 Thanks!